src/pages/RocketsCards.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  getRocket,
+  reserveRocket,
+  unreserveRocket,
+} from '../redux/rockets/rockets';
+
+const RocketsCards = () => {
+  const dispatch = useDispatch();
+  const fetchApi = 'https://api.spacexdata.com/v3/rockets';
+
+  const myRocketArray = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    const apiRockets = async () => {
+      const fetchRockets = await fetch(fetchApi);
+      const rockets = await fetchRockets.json();
+      return dispatch(getRocket(rockets));
+    };
+    if (myRocketArray.length === 0) {
+      apiRockets();
+    }
+  }, []);
+
+  const reserveRockets = (e) => {
+    dispatch(reserveRocket(e.target.id));
+  };
+
+  const unReserveRockets = (e) => {
+    dispatch(unreserveRocket(e.target.id));
+  };
+
+  return (
+    <div className="rocket-card">
+      {myRocketArray.map((rocket) => (
+        <div key={rocket.id} className="rockets-div">
+          <div className="rocket-imgs">
+            <img
+              src={rocket.flickr_images}
+              alt={rocket.rocket_name}
+              width="300"
+              height="200"
+              className="header-logo"
+              // eslint-disable-next-line react/jsx-no-duplicate-props
+              className="rocket-img"
+            />
+          </div>
+          <div className="rockets-desc">
+            <h3 className="rockets-desc-title">{rocket.rocket_name}</h3>
+            <p className="rockets-desc-p">
+              {rocket.reserved ? <button type="button" className="reserved-span">Reserved</button> : null}
+              {rocket.description}
+            </p>
+            {rocket.reserved ? (
+              <button
+                type="button"
+                onClick={unReserveRockets}
+                id={rocket.id}
+                className="unreserve-btn"
+              >
+                Cancel Reservation
+              </button>
+            ) : (
+              <button
+                type="button"
+                className="rockets-desc-btn"
+                onClick={reserveRockets}
+                id={rocket.id}
+              >
+                Reserve Rocket
+              </button>
+            )}
+          </div>
+        </div>
+      ))}
+    </div>
```

Comment:
As required, Kindly follow the design guide including the spacing and color 🙂 
> Your Rocket page design
![image](https://user-images.githubusercontent.com/66112760/151633197-3888a885-76ca-4ea3-854f-4ff15ac33d42.png)

> Requirement design wireframe
![image](https://user-images.githubusercontent.com/66112760/151633065-96c8db0b-16c4-4da9-92ee-8d1ddaf732eb.png)

Reply:
That has been changed